cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -1795,27 +1806,36 @@ private predicate flowFwdConsCand(
 
 pragma[nomagic]
 private predicate flowFwdIn(
-  DataFlowCall call, ParameterNode p, boolean fromArg, AccessPathOption argAp, AccessPathFront apf,
-  AccessPath ap, Configuration config
+  DataFlowCall call, ParameterNode p, CallContext innercc, CallContext outercc,
```

Comment:
I have a slight preference for the opposite order of the call context columns: "outer, inner" matches the direction of flow (and the column order in `pathIntoCallable`).